File: app/src/main/java/com/pluu/webtoon/item/Detail.java
package com.pluu.webtoon.item;

import java.util.List;

/**
 * 에피소드 상세 내용 Item Class
 * Created by nohhs on 2015-03-12.
 */
public class Detail {
	public String title;
	public String webtoonId, episodeId;
	public String nextLink, prevLink;
	public List<DetailView> list;

	// ERROR_TYPE
	public ERROR_TYPE errorType;
	public String errorMsg;
}

File: app/src/main/java/com/pluu/webtoon/item/Detail.kt
package com.pluu.webtoon.item

/**
 * 에피소드 상세 내용 Item Class
 * Created by pluu on 2017-05-02.
 */
class Detail {
    var title: String? = null
    var webtoonId: String? = null
    var episodeId: String? = null
    var nextLink: String? = null
    var prevLink: String? = null
    var list: List<DetailView>? = null

    // View Type
    var type = DETAIL_TYPE.DEFAULT

    // ERROR_TYPE
    var errorType: ERROR_TYPE? = null
    var errorMsg: String? = null
}
